src/components/bulk-email-tool/BulkEmailTool.jsx

```diff
@@ -7,29 +7,26 @@ import BulkEmailTaskManager from './bulk-email-task-manager/BulkEmailTaskManager
 import NavigationTabs from '../navigation-tabs/NavigationTabs';
 import BulkEmailForm from './bulk-email-form';
 import { CourseMetadataContext } from '../page-container/PageContainer';
+import { BulkEmailProvider } from './bulk-email-context';
 
 export default function BulkEmailTool() {
   const { courseId } = useParams();
-  const textEditorRef = useRef();
-  const copyTextToEditor = (body) => {
-    if (textEditorRef?.current) {
-      textEditorRef.current.setContent(body);
-    }
-  };
 
   return (
     <CourseMetadataContext.Consumer>
       {(courseMetadata) => (courseMetadata.isStaff ? (
         <div>
-          <NavigationTabs courseId={courseId} tabData={courseMetadata.tabs} />
-          <div>
-            <div className="row">
-              <BulkEmailForm courseId={courseId} cohorts={courseMetadata.cohorts} editorRef={textEditorRef} />
+          <BulkEmailProvider>
+            <NavigationTabs courseId={courseId} tabData={courseMetadata.tabs} />
```

Comment:
When we were going over this PR synchronously, I thought you mentioned that we may want to pull NavigationTabs out and be alongside/above the BulkEmailProvider. Just thought I may want to capture this in a comment in case it is something we need to change!

Reply:
Thank you for capturing that! I went ahead and made the change.